scripts/docker/docker-maker.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -ex
+
+CONTAINER_NAME=atomic-stagenet
+IMAGE_NAME=atomic-swap
+TAG=master
+
+# Setting NETWORK to "host" allows you to run swapcli commands on the local
+# host. You can also use "bridge", which requires all swapcli commands to
+# be run from inside the container.
+NETWORK=host
+
+# Note: We mount one directory above what swapd considers its "data-dir".
+DATA_MOUNT_DIR="${HOME}/.atomicswap/docker-maker"
+
+# Pre-create the mounted directory, or docker will create it with root
+# as the owner.
+mkdir -p "${DATA_MOUNT_DIR}"
+
+docker run --rm -v "${DATA_MOUNT_DIR}:/data" \
+	--env SWAPD_ENV=stagenet \
+	--env SWAPD_ETH_ENDPOINT="https://rpc.sepolia.org/" \
```

Comment:
should these be `dev`? i don't usually do specific maker/taker testing on stagenet

Reply:
if this is useful for you feel free to leave

Reply:
Sorry.  That was my own personal testing of the Dockerfile. I'll remove those scripts.